include/CoreGraphics/CGContext.h

```diff
@@ -261,4 +261,22 @@ COREGRAPHICS_EXPORT CGPoint CGContextConvertPointToUserSpace(CGContextRef c, CGP
 COREGRAPHICS_EXPORT CGSize CGContextConvertSizeToDeviceSpace(CGContextRef c, CGSize size);
 COREGRAPHICS_EXPORT CGSize CGContextConvertSizeToUserSpace(CGContextRef c, CGSize size);
 COREGRAPHICS_EXPORT CGRect CGContextConvertRectToDeviceSpace(CGContextRef c, CGRect rect);
-COREGRAPHICS_EXPORT CGRect CGContextConvertRectToUserSpace(CGContextRef c, CGRect rect);
\ No newline at end of file
+COREGRAPHICS_EXPORT CGRect CGContextConvertRectToUserSpace(CGContextRef c, CGRect rect);
+
+// [WinObjC Extension]
+// CGContextIwEnableEnhancedErrorHandling allows a developer to turn on advanced context error
+// handling. A CGContext that encounters errors when rendering will, by default, assert and abort.
+// Enhanced handling both silences asserts and enables an interested consumer to determine whether a
+// context has encountered a fatal drawing error.
+// A developer using enhanced error handling MUST check the status of a completed set of drawing
+// operations using CGContextIwGetError() and determine whether to redraw the frame.
+COREGRAPHICS_EXPORT void CGContextIwEnableEnhancedErrorHandling(CGContextRef context);
```

Comment:
maybe we should have some tag to indicate we have an extension?

Also it would make it easier to look at all the extensions we support/have